eyes-two/proguard-rules.pro

```diff
@@ -64,6 +64,3 @@
 # -keep public interface mypackage.MyInterface
 # -keep public class * implements mypackage.MyInterface
 
--keep class com.googlecode.eyesfree.textdetect.HydrogenTextDetector {
-    private long mNative;
```

Comment:
We don't, because HydrogenTestDetector pass pointer in methods: https://github.com/rmtheis/tess-two/blob/master/eyes-two/src/main/java/com/googlecode/eyesfree/textdetect/HydrogenTextDetector.java#L242